orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisActiveExecutionsMonitor.kt

```diff
@@ -56,13 +54,12 @@ class RedisActiveExecutionsMonitor(
   private val objectMapper: ObjectMapper,
   private val registry: Registry,
   @Value("\${queue.monitor.activeExecutions.refresh.frequency.ms:60000}") refreshFrequencyMs: Long,
-  @Value("\${queue.monitor.activeExecutions.cleanup.frequency.ms:300000}") cleanupFrequencyMs: Long
+  @Value("\${queue.monitor.activeExecutions.cleanup.frequency.ms:300000}") cleanupFrequencyMs: Long,
+  @Value("\${queue.monitor.activeExecutions.key:monitor.activeExecutions}") val redisKey: String
```

Comment:
Did you check that `monitor.activeExecutions` won't be resolved to a spring property? I only ask because I don't know SpEL behaviors.